packages/models/src/lib/implementation/schemas.ts

```diff
@@ -7,7 +7,12 @@ import {
 } from './limits';
 import { filenameRegex, slugRegex } from './utils';
 
-export const primitiveValueSchema = z.union([z.string(), z.number()]);
+export const primitiveValueSchema = z.union([
```

Comment:
I think it would be better to rename `primitiveValueSchema` to `tableCellValueSchema`, since that better describes it's purpose.

Reply:
done